homeassistant/components/recorder/migration.py

```diff
@@ -486,6 +486,34 @@ def _apply_update(engine, session, new_version, old_version):
         start = now.replace(minute=0, second=0, microsecond=0)
         start = start - timedelta(hours=1)
         session.add(StatisticsRuns(start=start))
+    elif new_version == 20:
+        # This changed the precision of statistics from float to double
+        if engine.dialect.name == "mysql":
+            _modify_columns(
+                connection,
+                engine,
+                "statistics",
+                [
+                    "mean DOUBLE",
+                    "min DOUBLE",
+                    "max DOUBLE",
+                    "state DOUBLE",
+                    "sum DOUBLE",
+                ],
```

Comment:
`DOUBLE PRECISION` should work for mysql as well. Is there still a need to define this separately?

references:
mysql: https://dev.mysql.com/doc/internals/en/floating-point-types.html
mariadb: https://mariadb.com/kb/en/double/

Reply:
Thanks, fixed as you suggest